publications/abstracts/reservoir-computing/background.tex

```diff
@@ -0,0 +1,22 @@
+\section{Background}
+
+Reservoir computing (RC) is a modern machine learning algorithm that enables
```

Comment:
Add RC to acronyms or don't use it as an acronym. Since RC is taken (radio control), I'd say, just use "reservoir computing" instead of increasing the number of acronyms the reader must remember.